app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/SortCode.scala

```diff
@@ -23,6 +23,16 @@ final case class SortCode(value: String) extends AnyVal
 
 object SortCode {
 
+  def withoutHyphensAndSpaces(maybeSortCode: String): SortCode =
+    SortCode(maybeSortCode.replaceAll("[-( )]+", ""))
+
+  def isValid(maybeSortCode: String): Boolean = {
+    val strippedSortCode: String = maybeSortCode.replaceAll("""[-( )]+""", "")
```

Comment:
And please, do not repeat code!
One day we all gonna be gone, and somebody comes in and will modify one regex, and forget about the other.
(but see my comment above, because in this case it is also unnecessary)